.github/workflows/sync-translated-content.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         run: |
           cd $GITHUB_WORKSPACE
           git remote add upstream "https://${GITHUB_ACTOR}:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git"
-          git config --local user.email "actions@users.noreply.github.com"
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
           git config --local user.name "MDN"
```

Comment:
Could you also use `GitHub <noreply@github.com>` here, or would that prevent filtering those commits?

Reply:
Just tested, if we use email `noreply@github.com`, the GitHub user will be @web-flow.

See this commit: b2d6c9f5112fc239c353f7f89e20f982e6875bc5 in [`t` branch](https://github.com/yin1999/translated-content/tree/t)

And the username will be override by github username if the email is binded by a user (as primary email).

Reply:
friendly pin @teoli2003, for a similar PR in content (mdn/content#20022)

Reply:
That's a good argument for using `github-actions[bot]`. What do you think @teoli2003?

Reply:
It seems that, we can just use `github-actions[bot] github-actions[bot]@users.noreply.github.com` for now. See: pypa/pipenv#4328.